config_defaults.yml

```diff
@@ -78,6 +79,13 @@ Gemfile:
       - gem: master_manipulator
       - gem: beaker-hostgenerator
         from_env: 'BEAKER_HOSTGENERATOR_VERSION'
+
+MAINTAINERS:
+  maintainers:
+    - github: puppetlabs
+      email: "discuss-modules@puppet.com"
```

Comment:
@ferventcoder has mentioned previously we should obsfucate the email to reduce spamming that account